packages/automerge-repo-storage-redis/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@automerge/automerge-repo-storage-redis",
+  "version": "1.0.0",
+  "description": "Redis storage adapter for Automerge Repo",
+  "repository": "https://github.com/automerge/automerge-repo/tree/master/packages/automerge-repo-storage-redis",
+  "author": "Ryan Kois <ryan.kois@gmail.com>",
+  "license": "MIT",
+  "type": "module",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w"
+  },
+  "dependencies": {
+    "@automerge/automerge-repo": "workspace:*",
+    "debug": "^4.3.4",
+    "ioredis": "^5.3.2"
```

Comment:
It feels like the `ioredis` must be peer dependency. What do you think?

Reply:
Yeah that makes sense, I agree